third-party/fbson/FbsonWriter.h

```diff
@@ -32,6 +32,8 @@
 #include "FbsonDocument.h"
 #include "FbsonStream.h"
 
+#pragma warning(disable : 4244)
```

Comment:
This should probably be re-enabled at the end of the header, shouldn't it?

Reply:
This is a template, so the warnings are emitted at the point of instantiation. so can not do that. A better idea would be great. At this point I decided not a big deal"

`third-party\fbson\FbsonJsonParser.h:#include "FbsonWriter.h"
third-party\fbson\FbsonWriter.h:#ifndef FBSON_FBSONWRITER_H
third-party\fbson\FbsonWriter.h:#define FBSON_FBSONWRITER_H
third-party\fbson\FbsonWriter.h:#endif // FBSON_FBSONWRITER_H
utilities\document\json_document_builder.cc:#include "third-party/fbson/FbsonWriter.h"`